<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mutilStringSend</class>
 <widget class="QWidget" name="mutilStringSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
background-color: rgb(255, 255, 255);
border-style:solid;
border-width:1px;
border-color:rgb(240,240,240);
border-radius:4px;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_windowTitle" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent; </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_windowTitle">
             <property name="toolTip">
              <string/>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:bold;
font-size:16px;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>多字符串界面</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>623</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_menu" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>45</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_close">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color:rgb(242, 113, 113); 
image: url(:/png/closeW.png)15 0 0 0;
}
QPushButton { 
image: url(:/png/close.png)15 0 0 0;
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
}
QPushButton:hover{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.08);
}
QPushButton:pressed{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.2);
}
QLineEdit{color: rgb(104, 112, 130);
    font-size:14px;
    border-style:solid;
    border-width:1px;
    border-height：20px;
    border-color:rgb(229, 231, 234);
    border-radius:4px;
    padding-left:5px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>349</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_importDb">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>导入</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_exportDb">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>导出</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_1">
            <attribute name="title">
             <string>示例</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>+</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea_stringShow">
           <property name="styleSheet">
            <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
border-style:solid;
border-width:1px;
border-color:rgb(240,240,240);
border-radius:4px;
}
QPushButton {
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
}
QPushButton:hover{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.08);
}
QPushButton:pressed{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.2);
}
QLineEdit{color: rgb(104, 112, 130);
    font-size:14px;
    border-style:solid;
    border-width:1px;
    border-height：20px;
    border-color:rgb(229, 231, 234);
    border-radius:4px;
    padding-left:5px;
}</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>740</width>
              <height>481</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
